### ISSUE TITLE
Missing Test Case - 2923. Find Champion I

### ISSUE THREAD

**Issue by Yawn-Sean** [LeetCode-Feedback/LeetCode-Feedback#17804]
#### Your LeetCode username
Yawn_Sean

#### Category of the bug
- [ ] Question
- [ ] Solution
- [ ] Language
- [x] Missing Test Cases 

#### Description of the bug

Using numbers in grid[i][i]

the test case is `[[1,1],[0,1]]`.

#### Code you used for Submit/Run operation

```
class Solution {
public:
    int findChampion(vector<vector<int>>& grid) {
        int n = grid.size();
        for (int i = 0; i < n; i++) {
            bool bad = 0;
            for (int j = 0; j < n; j++) {
                if (grid[j][i]) bad = 1;
            }
            if (!bad) return i;
        }
        return -1;
    }
};
```

#### Language used for code

C++

#### Expected behavior

0 while the code gives -1

#### Screenshots

![image](https://github.com/LeetCode-Feedback/LeetCode-Feedback/assets/106463906/2c97185c-ebdd-4753-b766-84961e3b7aee)

**Comment by LC-Pam** [comment]
Hi there,

Thank you for reaching out to us.

 We've relayed this issue to our team to investigate. 

We will reach back to you after we have the results.

Thank you for your support.

**Comment by projecteulerlover** [comment]
I disagree with this input. How can there be 1's on the main diagonal? A team cannot beat itself (and if it does, the problem requires additional clarification). Should be noted that all the inputs have 0's on the main diagonal, not 1's.

**Comment by Yawn-Sean** [comment]
> I disagree with this input. How can there be 1's on the main diagonal? A team cannot beat itself (and if it does, the problem requires additional clarification). Should be noted that all the inputs have 0's on the main diagonal, not 1's.

Actually, if you read the description carefully, you will find that there is just no constraints for the elements on the main diagonal. Also, the problem already told you that only the elements not on the main diagonal represents whether i beats j, so even if grid[i][i] == 1, it means nothing.

**Comment by projecteulerlover** [comment]
Sure, but at this point we can just say if this test case gets added, the LC contest team are actually weaponizing incompetence.

**Comment by Yawn-Sean** [comment]
> Sure, but at this point we can just say if this test case gets added, the LC contest team are actually weaponizing incompetence or just plain malicious.

I'm just providing a valid case ... And in fact, this case is valid.

Whether the case is added is up to them, and I believe it is OK for them to take either action (as for one thing, the description allows this to happen, and people like me check it many times; for another, this may be just a detail some people didn't notice) and it is up for them to decide.

**Comment by projecteulerlover** [comment]
That's a good point (and why I have removed that comment). 

I also see that this solution is from the current rank 1, so I can't exactly blame you for submitting this test case. Best of luck on the upcoming competitions :)

**Comment by LC-Pam** [comment]
Thank you for your time. 

We've used your feedback to check the problem. 

The team decided to reject this feedback after the evaluation. 

If you have any other questions or feedback, please don't hesitate to let us know! 

We appreciate your support!
